### PR TITLE
TASK: helper script to help setup e2e tests locally

### PR DESCRIPTION
**Pull request by dimaip**
A lot of people complained that E2E tests are hard to setup locally, so I created this script.

It has to go to 2.x, so the script would also be there.

### CODE REVIEW

**Comment by dimaip** [comment]
Need to rebase

**Comment by christophengelmayer** [comment]
You could fetch the code and specify the branch in a single command:

```
# fetch the test distribution in temporary folder (specify a specific branch using -b if you want)
git clone -b 5.0 https://github.com/neos/neos-ui tmp-neos-ui
mv tmp-neos-ui/Tests/IntegrationTests/TestDistribution neos-ui-test-distribution
rm -rf tmp-neos-ui
cd neos-ui-test-distribution

# create a database
mysql -u root -pnot_a_real_password --execute='CREATE DATABASE `neos`' # or whatever you want to call it
vi Configuration/Settings.yaml # adjust to the database that you have created
./setup.sh

# open a new tab and:
cd Packages/Application/Neos.Neos.Ui && make test-e2e

```

File: README.md [hunk 1]
```diff
@@ -123,16 +116,14 @@ on how to write meaningful descriptions for your contributions.
 
 #### Doing upmerges
 
-1) Develop only in the `2.x` branch, unless this feature is only compatible with Neos 4.x, then in `master`
-
-2) To do the upmerge do the following commands
+To do the upmerge run the following commands
```
Comment (markusguenther) [inline README.md]: I am fine with this PR but maybe we should explain more why we need to handle it this way. Or should we put more explanation on docs.neos.io for instance?

**Comment by markusguenther** [comment]
Already archived the test distribution. https://github.com/neos/neos-ui-testing-distribution

File: README.md (hunk 1 above)
Comment (dimaip) [reply README.md]: I'm just down merging the readme from master here, it's not a part of what I changed, can be adjusted later.

**Pull request merged**